libs/platform/src/lib/components/search-field/search-field.component.ts

```diff
@@ -282,6 +295,9 @@ export class SearchFieldComponent extends BaseComponent implements OnInit, OnDes
                     this._suggestionkeyManager.onKeydown($event);
                 }
                 break;
+            case 'Escape':
```

Comment:
Isn't it better to move this kind of stuff to enum?

Reply:
using keyUtil from core and Keycodes from cdk now.